framework/grid/ActionColumn.php

```diff
@@ -85,6 +85,28 @@ class ActionColumn extends Column
      * ```
      */
     public $buttons = [];
+    /** @var array visibility conditions for each button. The array keys are the button names (without curly brackets),
+     * and the values are the conditions whether this button is visible.Can be boolean value or callback. Defaults to true.
```

Comment:
I'm suggesting the following wording. Do you like it?

`are the conditions whether this button is visible. Can be boolean value or callback. Defaults to true.` -> `are the boolean true/false or the anonymous function. The button will be shown, when its name is not specified in this array`